imap/mboxlist.c

```diff
@@ -1789,9 +1791,7 @@ EXPORTED int mboxlist_deletemailbox_full(const char *name, int isadmin,
                                          const char *userid,
                                          const struct auth_state *auth_state,
                                          struct mboxevent *mboxevent,
-                                         int checkacl,
-                                         int local_only, int force,
-                                         int keep_intermediaries, int silent)
+                                         int flags, int silent)
```

Comment:
We could probably add MBOXLIST_DELETE_SILENT to flags and then ditch the _full API and just pass that flag through from sync_support.

Reply:
done